### PR TITLE
Fixes #133 :Improved indentation and removed blank spaces to improve Code Quality

### LINKED ISSUE #133: Need to improve indentation and Code Quality of footer.html
Working on it

### PR DESCRIPTION
**Pull request by saarthakchats**
Fixes #133

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
+                        <li><a target="_self" href="http://twitter.com/fossasia"><i class="icon social_twitter"></i></a></li>
+                        <li><a target="_self" href="http://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
+                        <li><a target="_self" href="https://www.linkedin.com/groups/FOSSASIA-3762811"><i class="icon social_linkedin icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
+                        <li><a target="default" href="https://plus.google.com/108920596016838318216" ><i class="icon social_googleplus"></i></a></li>
+                    </ul>
+                </div>
+        </div>
```
Comment (akshatnitd) [inline _includes/footer.html]: One `</div>` missing

File: _includes/footer.html [hunk 2]
```diff
@@ -1,46 +1,47 @@
...
+                        <li><a target="_self" href="https://fossasia.org/donate/">Donate</a></li>
+                        <li><a class="inner-link back-to-top" href="#top">Back To Top</a></li>
+                    </ul>
                 </div>
-            </footer>
+                <div class="col-sm-7">
+                        <span class="lead color-heading">About FOSSASIA</span>
+                            <p> FOSSASIA develops Open Source software and hardware for conversational AIs, science and event management with a global developer community from its base in Asia. The organization organizes Open Technology events, runs coding programs and the <a target="_self" href="https://codeheat.org" style="color:#777777">#Codeheat</a> development contest. The annual FOSSASIA Summit is the premier Open Technology event in Asia for developers, contributors, start-ups, and technology companies. FOSSASIA was founded in 2009 by <a target="_self" href="http://twitter.com/mariobehling" style="color:#777777">Mario Behling</a> and <a target="_self" href="http://twitter.com/hpdang" style="color:#777777">Hong Phuc Dang</a>.</p>
```
Comment (vaibhavsingh97) [inline _includes/footer.html]: `<p>` and `<span>` should be in the same line

File: _includes/footer.html [hunk 3]
```diff
@@ -1,46 +1,47 @@
...
+                        <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
+                        <li><a target="default" href="https://plus.google.com/108920596016838318216" ><i class="icon social_googleplus"></i></a></li>
+                    </ul>
+                </div>
+        </div>
+    </footer>           
+</div>
+
```
Comment (vaibhavsingh97) [inline _includes/footer.html]: Remove one blank line from here

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
LGTM!

File: _includes/footer.html (hunk 1 above)
Comment (vaibhavsingh97) [reply _includes/footer.html]: nice catch 😅 Thanks

**Comment by saarthakchats** [comment]
@vaibhavsingh97

**Comment by vaibhavsingh97** [comment]
@saarthakchats add div before `footer`

**Comment by saarthakchats** [comment]
@vaibhavsingh97

**Comment by saarthakchats** [comment]
@harshit98

File: _includes/footer.html [hunk 4]
```diff
@@ -1,46 +1,47 @@
...
+                        <li><a target="_self" href="http://facebook.com/fossasia"><i class="icon social_facebook"></i></a></li>
+                        <li><a target="_self" href="https://www.linkedin.com/groups/FOSSASIA-3762811"><i class="icon social_linkedin icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.youtube.com/fossasiaorg"><i class="icon social_youtube icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.flickr.com/photos/fossasia"><i class="icon social_flickr icon-large"></i></a></li>
+                        <li><a target="_self" href="https://www.instagram.com/explore/tags/fossasia/"><i class="icon social_instagram"></i></a></li>
+                        <li><a target="default" href="https://plus.google.com/108920596016838318216" ><i class="icon social_googleplus"></i></a></li>
+                    </ul>
+                </div>
+        </div>
+        </div>
```
Comment (nzec) [inline _includes/footer.html]: Fix the indentation here.

**Comment by saarthakchats** [comment]
@duttaditya18

**Review by nzec — APPROVED** [review]
Cool!

**Comment by arjunpandey2003** [comment]
Actually here you have left some whitespace
screen shot 2018-01-03 at 12 41 41 pm
The span and p should be inline with the ul tag not the li

**Comment by saarthakchats** [comment]
@arjunpandey2003 
Please specify where?

**Comment by arjunpandey2003** [comment]
17-18 line

**Comment by saarthakchats** [comment]
I DON'T SEE A PROBLEM

**Comment by arjunpandey2003** [comment]
@saarthakchats in the 18-19 line you have indented more as in left more space, causing it to be in-line with the li in the 23-27 line, instead the tags should come in-line with the ul on line 22

**Comment by arjunpandey2003** [comment]
@saarthakchats Perfect

**Comment by saarthakchats** [comment]
@arjunpandey2003 I believe its fine now

**Comment by arjunpandey2003** [comment]
@saarthakchats sorry i spotted one more problem, line 45, Check it out

**Review by harshit98 — APPROVED** [review]
LGTM 💯

**Comment by saarthakchats** [comment]
@harshit98 Can my task be approved?

**Pull request merged**